meta-lmp-base/recipes-containers/docker/files/0001-Fsync-layer-once-extracted-to-file-system.patch

```diff
@@ -1,4 +1,4 @@
-From 096f01b0496fdbccea611d9ad8ac38a9f389afad Mon Sep 17 00:00:00 2001
+From 8e55e79fe5d11776b0458f473c96ab7b2038c735 Mon Sep 17 00:00:00 2001
```

Comment:
Does this patch needs an "upstream-status"?

Reply:
Fixed